http-client.ts

```diff
@@ -128,28 +132,20 @@ export class HttpClient implements Server.IHttpClient {
```

Comment:
Consider renaming `theError` to `err` - it's not a better name, but it'll allow for cleaner code IMO